README.md

```diff
@@ -1,5 +1,15 @@
 # loopback-filters
 
+**⚠️ LoopBack 3 is in Maintenance LTS mode, only critical bugs and critical
+security fixes will be provided.**
+
+We urge all LoopBack 3 users to migrate their applications to LoopBack 4 as
+soon as possible. Refer to our
+[Migration Guide](https://loopback.io/doc/en/lb4/migration-overview.html)
+for more information on how to upgrade.
+
```

Comment:
Do we wanna add the LTS schedule like https://github.com/strongloop/generator-loopback/pull/415/files? Thanks.

Reply:
I wouldn't bother with that. This modules is feature incomplete and out of sync with LoopBack in many edge cases. Surprisingly enough, it has over 23k downloads per month 😱 https://www.npmjs.com/package/loopback-filters

Reply:
Never mind, I'll add the LTS section for consistency.

Reply:
Done, LGTY now?